tests/tcp-server/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net"
+)
+
+func main() {
```

Comment:
@vepatel @ciarams87 @lucacome @pleshakov There's a question for the team here. I've added a little tcp server to respond with it's IP. I'll be re-using it for the other TransportServer tests, for checking the lb method and such. 

What's the best way to engineer this?
Do we do a once off push of the docker image when we make a change to this server?
Do we add a pipeline/workflow?
Do we put it in a different repo?

Reply:
+1 for once-off push in a public repo and then pull it as usual through manifest

Reply:
I've added a small readme. I'm not sure where to push the image so it can be used
tests/tcp-server/README.md

Reply:
docker-hub public repo?

Reply:
done
`seanoneillf5/tcp-server:1.0`